### PR TITLE
Formatted earnings

### PR DESCRIPTION
**Pull request by mihoward21**
- drops all of the formatted earnings functions
- drops all of the non-formatted earnings types
- replaces all the regular earnings functions to use the formatted earnings types

still need to test

### CODE REVIEW

**Comment by mihoward21** [comment]
ok did some testing, think it's in a good state

File: packages/splits-sdk/src/subgraph/index.ts [hunk 1]
```diff
@@ -287,7 +287,7 @@ export const formatGqlAccount: (arg0: GqlAccount) => IAccountType = (
   }
 }
 
-const SPLITS_GRAPHQL_URL = 'http://api.splits.org/graphql'
+const SPLITS_GRAPHQL_URL = 'https://api.splits.org/graphql'
```
Comment (r0ohafza) [inline packages/splits-sdk/src/subgraph/index.ts:290]: ouch

**Review by r0ohafza — APPROVED** [review]
LGTM! thank you 🙏🏼

File: packages/splits-sdk/src/subgraph/index.ts (hunk 1 above)
Comment (mihoward21) [reply packages/splits-sdk/src/subgraph/index.ts:290]: hah, i think cloudfront is setup to forward any http requests to https, so it doesn't actually matter

**Pull request merged**